src/Fixer/Operator/UnaryOperatorSpacesFixer.php

```diff
@@ -75,4 +131,19 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             }
         }
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition()
+    {
+        return new FixerConfigurationResolver(
+            [
+                (new FixerOptionBuilder('not_operator_space', 'Space around logical `!` (`not`) operators.'))
```

Comment:
Wouldn't it be better to have an `operators` option supporting all possible operators like `binary_operator_spaces` has?